webview/src/shared/variables.scss

```diff
@@ -3,7 +3,7 @@ $bg-color: var(--vscode-editor-background);
 $bullet-size: 9px;
 $watermark-color: var(--vscode-descriptionForeground);
 
-$border-color: var(--vscode-dropdown-border);
+$border-color: var(--checkbox-border);
```

Comment:
[Q] Any ideas why some of these aren't prefixed with vscode?

Reply:
Just realized this actually 😄, I'll update this variable with the vscode prefixed version in the follow-up.